GOVERNANCE.md

```diff
@@ -0,0 +1,149 @@
+  with the project's coding and styling guidelines.
+
+* Sign the Individual CLA, or have your organization sign the Corporate CLA.
+
+* Submit your code as a PR with the appropriate DCO sign-off.
+
+## Committers
+
+Project Committers have merge access on the Rez GitHub repository
+and are responsible for approving submissions by Contributors.
+
+### Committer Responsibilities
+
+Typical activities of a Committer include:
+
+* Helping users and novice contributors.
+
+* Ensuring a response to questions posted to the ASWF rez slack channel.
+
+* Contributing code and documentation changes that improve the project.
+
+* Reviewing and commenting on issues and pull requests.
+
+* Ensuring that changes and new code meet acceptable standards and are
+  in the long-term interest of the project.
+
+* Participation in working groups.
+
+* Merging pull requests.
+
+### How to Become a Committer
+
+Any existing Committer can nominate an individual making significant
+and valuable contributions to the Rez project to become a new
+Committer.  New committers are approved by vote of the TSC.
+
+If you are interested in becoming a Committer, contact the TSC at ???
+
+TODO: best way to contact the TSC? Publicly? Privately?
+
+## Technical Steering Committee
+
+The Technical Steering Committee (TSC) oversees the overall technical
+direction of Rez, as defined in the project
+[charter](ASWF/charter/Rez-Technical-Charter.md).  This
+charter defines the TSC member terms and succession policies.
+
+TODO: Is there a rez charter? I don't know whether it belongs here or in a
+charter, but there should be some sort of mission statement that includes
+goals of increasing adoption of rez and growing the ecosystem of tools and the community of developers around rez.
+
+The responsibilities of the TSC include:
+
+* Coordinating technical direction of the project.
+
+* Project governance and contribution policy.
+
+* GitHub repository administration.
+
+* Maintaining the list of additional Committers.
+
+* Appointing representatives to work with other open source or open
+  standards communities.
+
+* Discussions, seeking consensus, and where necessary, voting on
+  technical matters relating to the code base that affect multiple
+  projects.
+
+* Coordinating any marketing, events, or communications regarding the
+  project.
+
+The TSC elects a Chair person, who acts as the project manager,
+organizing meetings and providing oversight to project
+administration. The Chair is elected by the TSC.  The Chair also
+serves as the Rez representative on the Academy Software
+Foundation (ASWF) Technical Advisory Council (TAC). The chair
+represents the project at ASWF TAC meetings.
+
+### Current TSC Members
+
+* Allan Johns (chair, author) - Nvidia
+* Jean-Christophe Morin
+* Stephen Mackenzie
+* Thorsten Kaufmann
+* Brendan Abel - Walt Disney Imagineering
+* John Mertic - The Linux Foundation (TODO: is John a TSC member?)
```

Comment:
No, John isn't a TSC member technically.

Reply:
I would recommend dropping the company names for now, since it's silly to update this document just for employment purposes when virtually every TSC member is doing their rez work independent of their employment.

Reply:
I can be interesting to show the employment though for transparency purposes. We wouldn't want everyone of us to start to work for Nvidia without telling anyone :)

Reply:
I am leaning towards dropping them here (specifically as none of us represents their company in the TSC). That said i agree we should be transparent about it. Is there another place we could make this transparent without persisting our employer history in the repo ;)

Reply:
I keep it maintained in the meeting notes, and would update them in whatever future meeting notes (as well as the template) when employment changes, as long as each TSC member lets us/me know. In this way, there will be a record of who worked for what company as time moves on across the notes, which I think is more accurate honestly. (Who you worked for at the time of various contributions)

Reply:
Seems good to me, my vote goes to no Employers here and Employers in meeting notes

Reply:
I know @instinct-vfx wasn't present during this topic, but I'll restate my suggestion here:

I am open to having the company names, because I think it gives transparency, but I think we should add a disclaimer/statement that various people aren't representing their companies on the TSC (as @maxnbk  says "doing their `rez` work independent of their employment".

While most TSC Governance docs do list companies but don't say anything else about it (OSL, OpenTimelineIO, OpenEXR, etc.), MaterialX has a little bit more clarity around company involvement:
* https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/GOVERNANCE.md

With that, I suggest something like (using Brendan as a hypothetical example (saying that Brendan's time is sponsored by Imagineering):

----
### Current TSC Members
Unless otherwise denoted as <sup>†</sup>, members of the TSC contribute their time independently of their employment.
* Allan Johns (chair, author) - Nvidia
* Brendan Abel - Walt Disney Imagineering <sup>†</sup>

Reply:
Thanks for following up, i'd be perfectly fine with this approach

Reply:
Also fine with this. If we can get consensus on this, I think it mostly wraps up items of discussion.

Reply:
consensus achieved, company names added, disclaimer string added, marking resolved